README.md

```diff
@@ -1,42 +1,59 @@
-# Rarible Protocol's JavaScript SDK Interface
-### An easy-to-use interface that allows LazyMint, Buying, Selling, Bidding and relevant methods from the [Protocol Flow](https://docs.rarible.com/#protocol-flow). 
-Rarible Protocol  Interface
+# The official JavaScript SDK of the Rarible Protocol 
+
+## Overview
+
+With the Rarible SDK, you can mint, buy, sell, and bid on NFTs on Ethereum (ERC-721 and ERC-1155). [Protocol Flow](https://docs.rarible.com/#protocol-flow). 
 
 > Disclaimer: This is still a Work in progress, and no logic has yet been developed for the SDK. This interface will serve as a cornerstone to integrate with the functions relevant to end users / developers of the Rarible ecosystem. 
 
-## Overview
 
-With `todo-name:RaribleSDK.js`, you can mint, buy, sell, and bid on NFTs on Ethereum (ERC-721 and ERC-1155) by accessing the most versatile functions of the Rarible Protocol. This allows developers like you to quickly iterate and build solutions for their specific use cases.
+## Installation
 
-This interface aims to cover the entire [Protocol Flow](https://docs.rarible.com/#protocol-flow). 
+`npm i --save <name>`
 
-*TODO: Verify these are the right methods that we're exposing*
-1. Creating ERC721/1155 Asset Metadata and Calling the Mint function
-2. Asset Discovery
-3. Creating a sell order
-4. Accepting a buyer/bid order
-5. Order Discovery
-6. Matching Order
-7. Updating/Canceling An Order
+## Quickstart:
 
-Read more about the Docs Here
-- https://docs.rarible.com/
+### Mint an NFT
+
+```
+const rarible = new RaribleSDK();
+const myNFT = {
+    supply: 5,
+    metadata: {
+        name: "Inception",
+        description: "The beginning.",
+        image: "...png"
```

Comment:
Will this be a file path? If so, I think we should have a more specific example like: `/some/path/image.png` which also begs the question as to whether we want to support globbing.